### ISSUE TITLE
[Feature request] A setting to reorder the conjugation requirements

### ISSUE THREAD

**Issue by Type-kun** [baileysnyder/japanese-conjugation#7]
I find the app is really useful for practice, especially at beginner level, but I also find myself spending as much, if not more, time trying to parse the current conjugation requirements in any given task as I spend actually conjugating the verb, because the order (temporal - affirmative - polite) feels unnatural for whatever reason. Other person that I recommended the app expressed the same thing independently, so it might be worth thinking about. 

A way to change the order in the settings would solve it nicely.

**Comment by lolbinarycat** [comment]
agreed, "polite negative past" corrosponds directly to ma-sen-deshita

i think i'm going to try to dust off my extremely rusty js to make a fork with this, but i don't think i can manage making the setting (i haven't seriously programmed in js in a long time)

**Comment by lolbinarycat** [comment]
forked version is up on https://envs.net/~binarycat/jconj/

**Comment by creallfluharty** [comment]
Also agreed, though I'd add that it might be helpful to allow for verbs to to be configured separately from adjectives where politeness is mostly useful at the end.
